### PR TITLE
#163318609 Api endpoint for editing a party name

### PR DESCRIPTION
**Pull request by perfect090**
**What does this PR do?**
Edits a specific party name

**Description of Task to be completed**
Have the following endpoints working
PATCH /api/v1/parties/:id/name/

**How should this be manually tested?**
They can be manually tested, using POSTMAN after cloning the repository

**What are the relevant pivotal tracker stories?**
#163318609

### CODE REVIEW

File: coverage/Politico/app.js.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
+<!doctype html>
+<html lang="en">
```
Comment (JCanaks) [inline coverage/Politico/app.js.html]: @perfect090 Your coverage folder is still showing up pls handle this

**Comment by JCanaks** [comment]
I'm yet to see screenshots as I mentioned in previous PRs. Pls take note of feedbacks

File: src/middlewares/validator.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import Helper from '../helpers/helper';
```
Comment (JCanaks) [inline src/middlewares/validator.js]: handle the case where a name is edited to a 
screen shot 2019-01-30 at 1 37 24 pm
string with just spaces

**Comment by perfect090** [comment]
Test result on POSTMAN from feed back received
screenshot 9

**Pull request closed without merging**